.jitpack.yml

```diff
@@ -3,4 +3,4 @@ jdk:
 
 install:
   # Exclude dokkaJavadoc tasks to save build time.
-  - ./gradlew --no-daemon --stacktrace -x dokkaJavadoc publishToMavenLocal
+  - ./gradlew --no-daemon --stacktrace -x dokkatooGenerate publishToMavenLocal
```

Comment:
I believe that should be `docsJavadocJar` instead. `dokkatooGenerate` is just an aggregator task that `publishToMavenLocal` does not even depend on.

Reply:
The comment above also needs to be updated. While at it, please generalize its wording an say

> Exclude generation of docs artifacts to save build time.